### PR TITLE
Add Staging Workflow in Best Practice

### PR DESCRIPTION
**Pull request by DavidKang**
screenshot_2018-11-22 using the obs web ui best practice guide

### CODE REVIEW

File: xml/obs_best_practice_staging_workflow.xml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      <para>
+        An Admin should previously create the manager groups.
+        Those managers would be the maintainers of the staging projects
+      </para>
+    </note>
+  </sect2>
+  <sect2>
+    <title>Start using the Staging Workflow</title>
+    <para>
+      After creating the Staging, we can start using it. In this view, we could
```
Comment (bgeuken) [inline xml/obs_best_practice_staging_workflow.xml]: There are a few cases where we call it Staging and some others where it's Staging Workflow. We should decide for one them and use it everywhere.

File: xml/obs_best_practice_staging_workflow.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+<sect1 version="5.1" xml:id="staging_workflow_how_to"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+  <title>Staging Worlflow</title>
+  <para></para>
+  <sect2>
+    <title>Creating a Staging Workflow</title>
+    <para>
+      At the moment the Staging Workflow is only accessible by hand.
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: At the moment**,** the [...]

File: xml/obs_best_practice_staging_workflow.xml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        <para>
+          <emphasis>Ignored:</emphasis> Request excluded for this Staging.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect2>
+  <sect2>
+    <title>Configuring a Staging</title>
+    <para>
+      In this section we can create/delete staging projects and also change the
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: In this section**,** we [...]

File: xml/obs_best_practice_staging_workflow.xml [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+<sect1 version="5.1" xml:id="staging_how_to"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+  <title>Staging</title>
+  <para></para>
+  <sect2>
+    <title>Creating a Staging</title>
+    <para>
+      At the moment, the Staging is only accessible by hand.
```
Comment (ChrisBr) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
      At the moment, the Staging page is only accessible by hand.
```

File: xml/obs_best_practice_staging_workflow.xml [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+      "<emphasis>/staging_workflows/new?project=openSUSE:Factory</emphasis>"
+      The creation of a Staging will automatically create two staging projects
```
Comment (ChrisBr) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
      The creation of a Staging workflow will automatically create two staging projects
```

File: xml/obs_best_practice_staging_workflow.xml [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+      as a subproject of the main project. Before create we need to select a
+      group of managers, they will be in charge of assign requests to the
```
Comment (ChrisBr) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
      group of managers, they will be in charge of assigning requests to the
```

File: xml/obs_best_practice_staging_workflow.xml [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+      staging projects and also exclude requests from the Staging.
```
Comment (ChrisBr) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
      staging projects and also excluding requests from the Staging workflow.
```

File: xml/obs_best_practice_staging_workflow.xml [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+    </para>
+    <figure>
+     <title>Creating a Staging for openSUSE:Factory</title>
```
Comment (ChrisBr) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
     <title>Creating a Staging workflow for openSUSE:Factory</title>
```

File: xml/obs_best_practice_staging_workflow.xml [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+          <emphasis>Ignored:</emphasis>
+          List of requests excluded from this Staging.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect2>
+  <sect2>
+    <title>Configuration</title>
+    <para>
+      The configuration of a Staging is very intuitive. We are able to
```
Comment (ChrisBr) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
      It is possible to
```

If it is intuitive or not is not our responsibility to judge :smile: (and also not part of a documentation)

File: xml/obs_best_practice_staging_workflow.xml [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+     <title>Configuring a Staging</title>
+     <mediaobject>
+      <imageobject>
+       <imagedata fileref="staging_workflow_configuration.png"/>
+      </imageobject>
+     </mediaobject>
+    </figure>
+    <note>
+      <para>
+        Changing the Managers Group of a Stanging will automatically unassign
```
Comment (ChrisBr) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
        Changing the Managers Group of a Staging workflow will automatically unassign
```

File: xml/obs_best_practice_staging_workflow.xml [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+        the old group and assing the new group to the related staging projects.
```
Comment (ChrisBr) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
        the old group and assign the new group to the related staging projects.
```

File: xml/obs_best_practice_staging_workflow.xml (hunk 1 above)
Comment (ChrisBr) [reply xml/obs_best_practice_staging_workflow.xml]: I think just calling it ``Staging`` sounds weird. 

``After creating a Staging...`` <- what staging? Idk...

But fine with it for now ...

**Pull request merged**